README.md

```diff
@@ -1,71 +1,214 @@
 [![CircleCI](https://circleci.com/gh/pylon/streamp3.svg?style=shield)](https://circleci.com/gh/spokestack/spokestack-python)
 [![PyPI version](https://badge.fury.io/py/spokestack.svg)](https://badge.fury.io/py/spokestack)
 [![Coverage Status](https://coveralls.io/repos/github/spokestack/spokestack-python/badge.svg?branch=master)](https://coveralls.io/github/spokestack/spokestack-python?branch=master)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-[![Documentation Status](https://readthedocs.org/projects/spokestack/badge/?version=latest)](https://spokestack.readthedocs.io/en/latest/?badge=latest)
+[![GitHub stars](https://img.shields.io/github/stars/spokestack/spokestack-python?style=social)](https://github.com/spokestack/spokestack-python/stargazers)
+![GitHub watchers](https://img.shields.io/github/watchers/spokestack/spokestack-python?style=social)
+![Twitter Follow](https://img.shields.io/twitter/follow/spokestack?style=social)
+
+Welcome to Spokestack Python! This library is intended for developing general purpose [voice interfaces](https://en.wikipedia.org/wiki/Voice_user_interface). What is a general purpose voice interface you ask? In our minds, a general purpose voice interface is a voice interface that can be easily integrated with virtually any application that uses [Python](https://www.python.org/). This includes [raspberrypi](https://www.raspberrypi.org/) applications such as a traditional smart speaker to a [Django](https://www.djangoproject.com/) web application. Literally, anything built in [Python](https://www.python.org/) can be given a voice interface.
 
 ## Get Started
 
 ### System Dependencies
 
+There are some system dependencies that need to be downloaded in order to install spokestack via pip.
+
 #### macOS
 
-    brew install lame portaudio
+```shell
+brew install lame portaudio
+```
 
 #### Debian/Ubuntu
 
-    sudo apt-get install portaudio19-dev libmp3lame-dev
+```shell
+sudo apt-get install portaudio19-dev libmp3lame-dev
+```
+
+#### Windows
+
+We currently do not support Windows 10 natively, and recommend you install [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) with the Debian dependencies. However, if you would like to work on native Windows support we will glady accept pull requests.
+
+Another potential avenue for using `spokestack` on Windows 10 is from [anaconda](https://www.anaconda.com/). This is without support for Text To Speech (TTS) though due to the Lame dependency. PortAudio, on the other hand, can be installed via `conda`.
+
+```shell
+conda install portaudio
+```
 
-Add voice capability to your projects with one line.
+### Installation with pip
 
-    pip install spokestack
+Once system dependencies have been satisfied, you can install the library with the following.
+
+```shell
+pip install spokestack
+```
 
 ## Development
```

Comment:
Typically, a "development" section in a library readme is for people contributing to the library, not using it.

Reply:
what should I call this instead?

Reply:
I ended up calling it. "Using the Library"